packages/roosterjs-react/lib/pasteOptions/plugin/createPasteOptionPlugin.ts

```diff
@@ -151,6 +151,15 @@ class PasteOptionPlugin implements ReactEditorPlugin {
                         false /*applyCurrentFormat*/,
                         true /** pasteAsImage **/
                     );
+                    break;
+                case 'pasteOptionPasteAsTextAndLinks':
```

Comment:
Other options should also have this behavior. So no need to have this value separately.

Reply:
Will do git force to the last head and roll the changes for this branch, these commits should not be in this PR :)